pay_android/pubspec.yaml

```diff
@@ -68,10 +68,10 @@ dependencies:
   flutter_localizations:
     sdk: flutter
   pay_platform_interface: ^1.0.3
-  flutter_svg: ^1.0.0
+  flutter_svg: ^2.0.0+1
```

Comment:
Let's use ^2.0.5

Reply:
Fixed in 5bfa1a6389dd13c70f4b4982db7f9d02ffa176b3